requirements.txt

```diff
@@ -0,0 +1,12 @@
+gdown==5.2.0
+gradio==5.4.0
+ipython==8.12.3
+matplotlib==3.9.2
+numpy==2.1.2
```

Comment:
Thank you for the pr, I tested on my local machine, and using this version of numpy leads to
```
ERROR: Cannot install -r requirements.txt (line 10), -r requirements.txt (line 2), -r requirements.txt (line 4) and numpy==2.1.2 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested numpy==2.1.2
    gradio 5.4.0 depends on numpy<3.0 and >=1.0
    matplotlib 3.9.2 depends on numpy>=1.23
    tensorflow 2.15.0 depends on numpy<2.0.0 and >=1.23.5
```
At the moment we recommend using Google Colab, together with our colab file which includes all necessary dependency installations to help you get started.